core/controllers/reader_test.py

```diff
@@ -115,41 +116,47 @@ def setUp(self):
         self.enable_string_classifier = self.swap(
             feconf, 'ENABLE_STRING_CLASSIFIER', True)
 
-        # Reading YAML exploration into a dictionary.
-        yaml_path = os.path.join(os.path.dirname(os.path.realpath(__file__)),
-                                 '../tests/data/string_classifier_test.yaml')
-        with open(yaml_path, 'r') as yaml_file:
-            self.yaml_content = yaml_file.read()
-
         self.login(self.VIEWER_EMAIL)
         self.signup(self.VIEWER_EMAIL, self.VIEWER_USERNAME)
 
         # Load demo exploration.
-        self.exp_id = '0'
+        self.unused_exp_id = '0'
```

Comment:
Oh. I'll change it then. classifier_services_test still uses exploration id 16 for loading string_classifier_yaml. Could you verify that @anmolshkl ?